Tasks/scss-targets.js

```diff
@@ -30,9 +57,10 @@ function compile(source, destination, compatibleBrowsers) {
             ],
         }))
         .pipe(sourcemaps.init({ loadMaps: true }))
-        .pipe(sass({ linefeed: 'crlf' }).on('error', sass.logError))
+        .pipe(sass({ linefeed: process.platform === 'win32' ? 'crlf' : 'lf' }).on('error', sass.logError))
```

Comment:
Won't it pose a problem when the CSS file contains CRLF line endings, but the content in the source map contains LF line endings? Or are these independent of each other?

Reply:
They are independent from each other, this is perfectly safe.